src/main/java/com/jacob/duke/Ui.java

```diff
@@ -0,0 +1,150 @@
+package main.java.com.jacob.duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Scanner;
+import java.util.function.Predicate;
+
+import main.java.com.jacob.duke.task.Task;
+
+public class Ui {
+    /**
+     * UI of the Done command
+     * @param taskDescription The currently operated task's status
+     */
+    public void showDone(String taskDescription) {
+        System.out.println(" Nice! I've marked this task as done: \n" + taskDescription);
+    }
+
+    /**
+     * UI for the newly added tasks
+     * @param taskDescription The currently operated task's status
+     * @param taskList List representation of the current task list
+     */
+    public void showNewTaskAdded(String taskDescription, List<Task> taskList) {
+        System.out.printf(
+                " Got it. I've added this task: \n   %s\n Now you have %d tasks in the list.\n", taskDescription, taskList.size());
```

Comment:
Not a violation but you could follow the format of showTaskDeleted to improve readability!